### DISCUSSION TITLE
kimpanel 及对应的 gnome extension 能否提供方便的重启 fcitx 服务的入口？包括支持 flatpak 的 fcitx5

### DISCUSSION THREAD

**Discussion by cheese** [fcitx/fcitx5#476] [Ideas]
现在提供了 Reload Configuration，但不会重启服务。如果能重启服务甚至重启 flatpak 就方便用户在线更新了

**Comment by wengxt** [comment]
已经 https://github.com/fcitx/fcitx5/commit/9d0fad6d0c67d08fb4cb75bcffd3b1cba2297b85 增加针对 flatpak 的重启服务，然后把 kimpanel 的 reloadconfig 行为改成重启 https://github.com/fcitx/fcitx5/commit/db25d60b6ba2844a4eb94015fea1c8cfc3544997
